packages/Liquid-Core.package/LiquidAnswerPanel.class/instance/maxNumberOfChoices.st

```diff
@@ -0,0 +1,4 @@
+accessing
+maxNumberOfChoices
+
+	^ maxNumberOfChoices
```

Comment:
`maxNumberOfChoices` seems to be a method to place on the Question object 

```suggestion
	^ question maxNumberOfChoices
```

Reply:
So far, `SingleChoiceQuestions` do not implement `maxNumberOfChoices`, which means we cannot use your suggestion outright. Given that in the end we basically always convert a `SingleChoiceQuestion` to a `MultipleChoiceQuestion` with `maxChoice: 1` (in `question:` of `LiquidAnswerPanel`), I wanted to suggest removing the distinction between the two at tomorrow's meeting.

Reply:
Since that suggestion was accepted, we will implement it in another PR